### PR TITLE
Add H-60 coordinate reporting + keyboard extension 

### PR DESCRIPTION
**Pull request by SPAS79**
Added the parameters in the hook to make coordinate reporting consistent with the U/S/MH-60 *Hawk helicopter mod.

Added an extension .lua that draws a keyboard alongside the numeric keypad.

### CODE REVIEW

File: Scripts/Hooks/scratchpad-hook.lua [hunk 1]
```diff
@@ -603,6 +603,8 @@ local function loadScratchpad()
             return {DDM = true, MGRS = true}
         elseif string.sub(ac, 1, 4) == "F-14" then
             return {DDM = {precision = 1}}
+        elseif string.sub(ac, 2, 5) == "H-60" then
+            return {DDM = {precision = 2, lonDegreesWidth = 3}, MGRS = true}
```
Comment (rkusa) [inline Scripts/Hooks/scratchpad-hook.lua:607]: 👍

File: Scripts/Scratchpad/Extensions/Disabled/keyboard.lua [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+local keyBMatrix = {
+  {"Q", "W", "E", "R", "T", "Y", "U", "I", "O", "P"},
+  {"A", "S", "D", "F", "G", "H", "J", "K" ,"L", "⌫"},
+  {"Z", "X", "C", "V", "B", "N", "M", {["␣"] = " "}, ".", ","},
+} 
+
+
+--local width = 30
+--local height = 30
```
Comment (rkusa) [inline Scripts/Scratchpad/Extensions/Disabled/keyboard.lua:18]: Any particular reason to have them commented out?

**Review by rkusa — COMMENTED** [review]
Nice, thanks

File: Scripts/Scratchpad/Extensions/Disabled/keyboard.lua (hunk 2 above)
Comment (SPAS79) [reply Scripts/Scratchpad/Extensions/Disabled/keyboard.lua:18]: Not that I can remember of. I was probably just experimenting with keyboard keys.

**Comment by rkusa** [comment]
Sorry that it took so long. I was planing to give it a final look and merge it, but I think you continued additional work in your fork which also made it into the PR. If you don't mind, please resubmit a PR with just the H60 changes, and a separate one for the keyboard extension if you think it is worth adding in here. I'll close the PR in the meantime. Thanks

**Comment by SPAS79** [comment]
Hey there! 

Well the thing kind of evolved a little. 

I found Draken35's version (he made a similar keyboard to mine) which also loads coordinates automatically into some of the modules, and I have added the logic for the H-60 to that version. 

Now he doesn't want to have anything to do with github, but he gave me permission to mess around and publish a fork on github so I did. 

So now we have a version of DCS scratchpad which works in VR, picks up coordinates from the map and loads them in a plethora of compatible modules (including the H-60 series of submods). Yay! I am really happy with this and I have stopped using TheWay for the moment (I was forced to use scratchpad anyway as TheWay does not project any aiming device on the map).

The conundrum here is we should really ask him if he's ok with his code (mine works on his, which in turn I understand works on other people's work) being merged with yours, I guess? 

The link to draken35's original post is in my fork's readme.md

**Pull request closed without merging**